core/src/display_object.rs

```diff
@@ -1131,6 +1143,20 @@ macro_rules! impl_display_object_sansbounds {
         fn as_ptr(&self) -> *const crate::display_object::DisplayObjectPtr {
             self.0.as_ptr() as *const crate::display_object::DisplayObjectPtr
         }
+        fn sound_instances(&self) -> Vec<SoundInstanceHandle> {
+            self.0.read().$field.sound_instances.clone()
```

Comment:
I would like to avoid the `.clone()` here, but I'm not sure how.
Ideally this method needs to return `&Vec<SoundInstanceHandle>`.